single_tech_samples/storage/sample1_data_lifecycle/terraform/provider.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
```

Comment:
Once you make the other changes, this file won't be required so please remove it.

Reply:
Done